omnibus/config/software/python3.rb

```diff
@@ -38,6 +39,9 @@
   build do
     ship_license "PSFL"
 
+    # security patches
```

Comment:
Add a comment to remove this patch once python 3 is updated to 3.8.11, 3.9 or 3.10